app/views/CallView/index.tsx

```
@@ -1,24 +1,56 @@
-import React from 'react';
+import React, { useEffect } from 'react';
```

ESLint

[error] There should be no empty line within import group

(import/order)